index.html

```diff
@@ -19,7 +19,9 @@
 
 		<div ng-app="app">
 			<div ng-controller="TestController as test">
-				<rl-rich-text-editor ng-model="test.text"></rl-rich-text-editor>
+				Selection: {{test.selection}}
+				<rl-select ng-model="test.selection" options="test.options" option-as="item"
```

Comment:
Yeah, you would almost need to use a directive to alias the item in the ng-repeat, sort of like what we do with card container cards